client/web/src/site-admin/smtp/SendTestEmailForm.tsx

```diff
@@ -74,7 +86,7 @@ export const SendTestEmailForm: FC<SendTestEmailProps> = ({ authenticatedUser, c
                 name="authentication"
                 message={
                     <>
-                        Verify saved configuration by choosing an email already{' '}
+                        Verify your configuration by choosing an email already{' '}
```

Comment:
```suggestion
                        Verify your configuration by choosing an email{' '}
```

The `already` is unnecessary.

Reply:
<img width="1241" alt="CleanShot 2023-08-22 at 11 09 08@2x" src="https://github.com/sourcegraph/sourcegraph/assets/25608335/422cec95-0c8d-4f52-843b-8de67325404d">

Yeah, Grammarly agrees lol.

Reply:
Good catch, I used the same copy as in the Figma designs, but you're right, it seems unnecessary :)